api/src/main/java/org/cardanofoundation/rosetta/common/services/impl/CardanoServiceImpl.java

```diff
@@ -368,22 +365,23 @@ public UnsignedTransaction createUnsignedTransaction(NetworkIdentifierType netwo
     String transactionBytes = HexUtil.encodeHexString(
         com.bloxbean.cardano.yaci.core.util.CborSerializationUtil.serialize(mapCbor));
     log.info("[createUnsignedTransaction] Hashing transaction body");
-    String bodyHash = HexUtil.encodeHexString(
+    String bodyHash = com.bloxbean.cardano.client.util.HexUtil.encodeHexString(
```

Comment:
Why you are adding here the name of the package? I believe that we have imported exactly the same package (as on the 372nd line)

Reply:
I will change, maybe auto_magic)